js/samples/06.auth.botSSO/README.md

```diff
@@ -0,0 +1,156 @@
+    - [Using Teams Toolkit for Visual Studio Code](#using-teams-toolkit-for-visual-studio-code)
+    - [Using Teams Toolkit CLI](#using-teams-toolkit-cli)
+    - [Manually upload the app to a Teams desktop client](#manually-upload-the-app-to-a-teams-desktop-client)
+  - [Deploy the bot to Azure](#deploy-the-bot-to-azure)
+  - [Further reading](#further-reading)
+
+<!-- /code_chunk_output -->
+
+## Setting up the sample
+
+1. Clone the repository
+
+    ```bash
+    git clone https://github.com/Microsoft/teams-ai.git
+    ```
+
+2. In the root JavaScript folder, install and build all dependencies
+
+    ```bash
+    cd teams-ai/js
+    yarn install
+    yarn build
+    ```
+
+3. In a terminal, navigate to the sample root.
+
+    ```bash
+    cd teams-ai/js/samples/01.messaging.a.SsoAuthBot/
+    yarn start
+    ```
+
+## Interacting with the bot
+
+You can interact with this bot by sending it a message, which will echo back to you.
+
+## Multiple ways to test
+
+The easiest and fastest way to get up and running is with Teams Toolkit as your development guide. To use Teams Toolkit to continue setup and debugging, please continue below.
+
+Otherwise, if you only want to run the bot locally and build manually, please jump to the [BotFramework Emulator](#testing-in-BotFramework-emulator) section.
+
+### Using Teams Toolkit for Visual Studio Code
+
+The simplest way to run this sample in Teams is to use Teams Toolkit for Visual Studio Code.
+
+1. Ensure you have downloaded and installed [Visual Studio Code](https://code.visualstudio.com/docs/setup/setup-overview)
+1. Install the [Teams Toolkit extension](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension)
+1. Select **File > Open Folder** in VS Code and choose this sample's directory from the repo
+1. Using the extension, sign in with your Microsoft 365 account where you have permissions to upload custom apps
+1. Select **Debug > Start Debugging** or **F5** to run the app in a Teams web client.
+1. In the browser that launches, select the **Add** button to install the app to Teams.
+
+> If you do not have permission to upload custom apps (sideloading), Teams Toolkit will recommend creating and using a Microsoft 365 Developer Program account - a free program to get your own dev environment sandbox that includes Teams.
+
+### Using Teams Toolkit CLI
+
+You can also use the Teams Toolkit CLI to run this sample.
+
+1. Install the CLI
+
+    ```bash
+    npm install -g @microsoft/teamsfx-cli
+    ```
+
+1. Open a second shell instance and run ngrok tunneling service - point to port 3978
+
+    ```bash
+    ngrok http --host-header=rewrite 3978
+    ```
+
+1. Copy the ngrok URL and put the URL and domain in the `/env/env.local` file
+
+    ```bash
+    BOT_ENDPOINT=https://{ngrok-url}.ngrok.io
+    BOT_DOMAIN={ngrok-url}.ngrok.io
+    ```
+
+1. In the repository directory, run the Teams Toolkit CLI commands to automate the setup needed for the app
```

Comment:
Please update readme content to match other samples - we are now redirecting some of the directions to global docs for sanity sake

Reply:
(Same for ME readme)

Reply:
There is a separate ticket (#783)  for improving the samples: Added this point to the ticket.